static/templates/subscription_settings.handlebars

```diff
@@ -16,9 +16,9 @@
                 <span class="checkmark" data-finish-editing=".stream-name-editable">✓</span>
                 {{/if}}
             </div>
-            <button class="button small rounded subscribe-button sub_unsub_button {{#unless subscribed }}unsubscribed{{/unless}}" type="button" name="button">
+            <button class="button small rounded subscribe-button sub_unsub_button {{#unless subscribed }}unsubscribed{{/unless}}" type="button" name="button" title="{{#unless subscribed }}{{t 'Subscribe to stream'}}{{/unless}}{{#if subscribed }}{{t 'Unsubscribe from stream'}}{{/if}} (S)">
```

Comment:
This has a small bug, which is that it doesn't update after you subscribe or unsubscribe. To reproduce, go to a stream, type `S`, and hover over the button.

Maybe the text could be "Toggle subscription (S)"?